src/pixfinder.d.ts

```diff
@@ -4,70 +4,90 @@ export interface Point {
   readonly y: number;
 }
 
-export interface FindAllFeaturesOptions {
-  // If accuracy = 1 then Pixfinder analyzes each pixel of the image, if accuracy = 2 then each 2nd pixel, and so on. 
-  // Large number for better performance and worse quality and vice versa. The number should be a positive integer.
-  // Default: 2
+export interface FindAllOptions {
+  /** If accuracy = 1 then Pixfinder analyzes each pixel of the image, if accuracy = 2 then each 2nd pixel, and so on. 
+    *  Large number for better performance and worse quality and vice versa. The number should be a positive integer.
+    *  Optional. (Default: 2)
+    */
   accuracy: number;
```

Comment:
The property itself should be marked as optional via "?", as it is stated in TypeScript documentation, otherwise, the TypeScript compiler will not treat it as optional and will throw warnings.